src/pages/gallery.astro

```diff
@@ -0,0 +1,181 @@
+          </div>
+          <div class="image-container" data-alt="UBC iGEM team group photo">
+            <img src="https://picsum.photos/400/450?random=27" alt="UBC iGEM team group photo" />
+          </div>
+          <div class="image-container" data-alt="Laboratory research and data analysis">
+            <img src="https://picsum.photos/400/280?random=28" alt="Laboratory research and data analysis" />
+          </div>
+          <div class="image-container" data-alt="Project presentation at competition">
+            <img src="https://picsum.photos/400/520?random=29" alt="Project presentation at competition" />
+          </div>
+        </div>
+
+        <!-- Column 3 -->
+        <div class="flow">
+          <div class="image-container" data-alt="iGEM competition presentation">
+            <img src="https://picsum.photos/400/480?random=30" alt="iGEM competition presentation" />
+          </div>
+          <div class="image-container" data-alt="Genetic engineering and cloning work">
+            <img src="https://picsum.photos/400/320?random=31" alt="Genetic engineering and cloning work" />
+          </div>
+          <div class="image-container" data-alt="STEM education and public engagement">
+            <img src="https://picsum.photos/400/580?random=32" alt="STEM education and public engagement" />
+          </div>
+          <div class="image-container" data-alt="Project planning and brainstorming">
+            <img src="https://picsum.photos/400/420?random=33" alt="Project planning and brainstorming" />
+          </div>
+        </div>
+
+        <!-- Column 4 -->
+        <div class="flow">
+          <div class="image-container" data-alt="Team receiving recognition and awards">
+            <img src="https://picsum.photos/400/380?random=14" alt="Team receiving recognition and awards" />
+          </div>
+          <div class="image-container" data-alt="Advanced laboratory techniques">
+            <img src="https://picsum.photos/400/550?random=15" alt="Advanced laboratory techniques" />
+          </div>
+          <div class="image-container" data-alt="Networking with other iGEM teams">
+            <img src="https://picsum.photos/400/360?random=16" alt="Networking with other iGEM teams" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+</Layout>
+
+<style>
+  h1 {
+    font-size: 4.5em;
+    font-weight: 900;
+    margin-top: 0.5em;
+    margin-bottom: 0.5em;
+    color: var(--color-primary-black);
+  }
+
+  .gallery-hero-section {
+    background-color: var(--color-primary-green);
+    padding-block: 5%;
+    padding-inline: max(10%, min(110px, 15vw));
+    text-align: left;
+    border-bottom: solid var(--border-width) var(--border-color);
+    outline: var(--border-color);
+    max-width: 100vw;
+    margin-left: calc(-1 * min(110px, 15vw));
+    margin-right: calc(-1 * min(110px, 15vw));
```

Comment:
???

- why use `padding-block` and `padding-inline` here? this is way more clearly expressed using a single `padding` declaration, and you're not even doing anything writing-directionality-related here
- why do you need *negative margin* here??